### DISCUSSION TITLE
Dense models has too many points

### DISCUSSION THREAD

**Discussion by paujar** [colmap/colmap#2219] [General]
Yes, this has happened. I have a big area, I mean big and I have generated point cloud from it with SIFT. But there are some gaps in there and I wanted to create same area with same photos (6387 images used) for DSP SIFT. It is working, but when fusing, it informs that it has 1.33 billion points. So reaaally heavy. The SIFT only process create 380 million points, that was ok. Is there a way to give hard limit to points? because now it's a bit lottery or is there something that could help

![image](https://github.com/colmap/colmap/assets/120630944/12316902-1616-488c-9418-690f0d479316)

**Comment by ahojnnes** [comment]
You can reduce the image resolution for dense reconstruction and/or adjust the fusion thresholds ro produce fewer points. Both should help but may have slightly different outcomes.
